polldevs.cf.example

```diff
@@ -0,0 +1,34 @@
+# List of routers for Zino to monitor.
+#
+# An empty line signifies the start of a new configuration block, i.e. a set of
+# defaults, or a new, distinct router.  Router names can be arbitrary, but must
+# be unique across all of Zino.  Comments begin with the # character.
+
+# Default values that will apply to all router entries unless overridden:
+default interval: 5
+default community: public
+default domain: example.org
+default hcounters: yes
```

Comment:
Would be nice to have more information about those default values (specifically interval and hcounters)

Reply:
What is the time unit for interval: seconds? minutes?

Reply:
minutes, yeah. It says so further down in the file, when `interval` is specified again, but I've added a comment to the example.